src/.nuget/dir.props

```diff
@@ -67,12 +67,8 @@
     <When Condition="'$(PackageRID)' != ''" />
     <When Condition="'$(_runtimeOSFamily)' == 'win'">
       <PropertyGroup>
-        <RIDPlatform>win7</RIDPlatform>
-        <RIDPlatform Condition="'$(ArchGroup)' == 'arm'">win8</RIDPlatform>
-        <RIDPlatform Condition="'$(ArchGroup)' == 'arm64'">win10</RIDPlatform>
-        
-        <!-- Set the platform part of the RID if we are doing a portable build -->
-        <RIDPlatform Condition="'$(PortableBuild)' == 'true'">win</RIDPlatform>
+        <!-- Note: Windows builds are always portable (-PortableBuild=false is ignored) -->
+        <RIDPlatform>win</RIDPlatform>
```

Comment:
(minor) I think you can just get rid of the `RIDPlatform` property all together.  I don't see it being used elsewhere.

```
<!-- Note: Windows builds are always portable (-PortableBuild=false is ignored) -->
<PackageRID>win-$(ArchGroup)</PackageRID>
```

Reply:
Good call.  For some reason I thought the other cases were setting that and I was being consistent.  Upon reflection, you are correct.